pkg/wsman/amt/ethernetport/settings.go

```diff
@@ -27,13 +26,15 @@ type (
 		XMLName           xml.Name     `xml:"Body"`
 		EthernetPort      EthernetPort `xml:"AMT_EthernetPortSettings"`
 		EnumerateResponse common.EnumerateResponse
+		PullResponse      PullResponse
 	}
 
 	EthernetPort struct {
```

Comment:
@zaidusmani26 I dug into this a bit more and the EthernetPort struct is missing a few more properties.  It should look like this so that it can handle both wired and wireless responses from AMT. 

EthernetPort struct {
		ElementName                  string
		InstanceID                   string
		VLANTag                      int
		SharedMAC                    bool
		MACAddress                   string
		LinkIsUp                     bool
		LinkPolicy                   []int
		LinkPreference               int
		LinkControl                  int
		SharedStaticIp               bool
		SharedDynamicIP              bool
		IpSyncEnabled                bool
		DHCPEnabled                  bool
		IPAddress                    string
		SubnetMask                   string
		DefaultGateway               string
		PrimaryDNS                   string
		SecondaryDNS                 string
		ConsoleTcpMaxRetransmissions int
		WLANLinkProtectionLevel      int
		PhysicalConnectionType       int
		PhysicalNicMedium            int
	}